File: mandysax/src/main/java/mandysax/Tools/ToastUtils.java
package mandysax.Tools;

import android.content.Context;
import android.widget.Toast;

public class ToastUtils
{
    private static Context con;

    public static void init(Context context)
	{
        con = context;
    }
    public static void showToast(CharSequence text)
	{
        if (text != null && con != null)Toast.makeText(con, text, 10).show();
    }

}

File: app/src/main/java/studio/mandysa/music/logic/utils/ToastUtils.kt
package studio.mandysa.music.logic.utils

import android.content.Context
import android.view.Gravity
import android.widget.Toast

fun Context.toast(msg: String) {
    val toast = Toast.makeText(this, msg, Toast.LENGTH_SHORT)
    toast.setGravity(Gravity.CENTER, 0, 0)
    toast.show()
}